config/projects.yml

```diff
@@ -191,8 +191,8 @@ doctrine-phpcr-admin-bundle:
       php: ['5.3', '5.4', '5.5', '5.6', '7.0', '7.1']
       versions:
         symfony: ['2.3', '2.7', '2.8']
-        sonata_admin: ['2']
```

Comment:
https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/blob/1.x/composer.json

1.x does not allow SonataAdmin 3.x versions. So this should not be changed.